diff --git a/backend/backend.c b/backend/backend.c
index bfb43ba0..fe8f923c 100644
--- a/backend/backend.c
+++ b/backend/backend.c
@@ -223,7 +223,7 @@ static struct wlr_backend *attempt_drm_backend(struct wl_display *display,
        struct wlr_backend *primary_drm = NULL;
        for (size_t i = 0; i < (size_t)num_gpus; ++i) {
                struct wlr_backend *drm = wlr_drm_backend_create(display, session,
-                       gpus[i], primary_drm);
+                       gpus[i], NULL);
                if (!drm) {
                        wlr_log(WLR_ERROR, "Failed to create DRM backend");
                        continue;

